numpy/core/code_generators/generate_ufunc_api.py

```diff
@@ -36,11 +36,14 @@
 static NPY_INLINE int
 _import_umath(void)
 {
-  PyObject *numpy = PyImport_ImportModule("numpy.core.umath");
+  PyObject *numpy = PyImport_ImportModule("numpy.core._multiarray");
```

Comment:
I'm very confused, shouldn't it be `_multiarray_umath` here and below?

Reply:
fixed, thanks